Extensions/InvariantCultureTypeConversionExtensions.cs

```diff
@@ -20,5 +20,14 @@ public static class InvariantCultureTypeConversionExtensions
         /// <returns>A predictable string representation of the given <paramref name="value"/>.</returns>
         public static string ToInvariantString(this DateTime value, string format) => 
             value.ToString(format, CultureInfo.InvariantCulture);
+
+        /// <summary>
+        /// Returns the string representation of the given <paramref name="value"/> using the <see
+        /// cref="CultureInfo.InvariantCulture"/>.
+        /// </summary>
+        /// <param name="value">The value to convert to a string.</param>
+        /// <returns>A predictable string representation of the given <paramref name="value"/>.</returns>
+        public static string ToInvariantString(this DateTime? value) =>
+            !value.HasValue ? string.Empty : value.Value.ToString(CultureInfo.InvariantCulture);

```

Comment:
Mit szólsz ehhez?
```suggestion
            value?.ToString(CultureInfo.InvariantCulture) ?? string.Empty;
```

Reply:
Sőt igazából, hívjuk az eredetit: 
```suggestion
            value?.ToInvariantString() ?? string.Empty;
```

Reply:
Valamint ha már hozzányúltunk akkor legyen meg a format paraméteresnek is a nullable párja.